ui/admin/view.php

```diff
@@ -182,24 +180,18 @@
 				}//end if
 
 				if ( 0 < count( $fields ) ) {
+					if ( $more ) {
+						$title = __( 'More Fields', 'pods' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 50c8b86e26838c1b8fb139b0eb625bc8</sub>